README.md

```diff
@@ -177,7 +177,7 @@ Note that the Docker stacks `*-notebook` images tagged `2d878db5cbff` include th
 docker pull jupyter/scipy-notebook:2d878db5cbff
 ```
 
-Note: If you choose to use a different container image, be sure to set the `DOCKER_CONTAINER_IMAGE` environment variable either in the shell you use to launch JupyterHub or in the `.env` file.
+Note: If you choose to use a different container image, you must set the `DOCKER_CONTAINER_IMAGE` environment variable in the `.env` file, or override it in the environment where you launch JupyterHub.
```

Comment:
``...override it in the environment where you launch JupyterHub.`` Clarify ``it`` as variable or file or either?